src/rcx/src/extmain.cpp

```diff
@@ -109,6 +109,7 @@ extMain::extMain()
 }
 extMain::~extMain()
 {
+  _search.reset();
```

Comment:
not needed, the dtor will clean up the memory

Reply:
Will the destructor run automatically if I don't delete the `_search`, or should I add a `delete _search`? The documentation isn't really clear on what will be the behaviour

Reply:
Class members are always destroyed when the class is destroyed.  See https://en.cppreference.com/w/cpp/language/destructor:
```
...the compiler calls the destructors for all non-static non-variant data members of the class, in reverse order of declaration,...
```